week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,16 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
```

Comment:
Here Laetitia you should have multiplied by 20%. Fix this part of the function please

Reply:
20% is also equals to a fifth I believe, that's why I put " / 5"